### DISCUSSION TITLE
Проксирование сайтов, запрещающие доступ из РФ

### DISCUSSION THREAD

**Discussion by dodedov** [anticensority/runet-censorship-bypass#155] [Ideas]
Привет! Цензуру вводит не только РКН, но пользователям из РФ разные сайты и сервисы.

На сколько возможно расширить этим функционалом ваш сервис?

Сейчас такие блокировки решаю Torом, добавленного в Свои прокси.

**Comment by ilyaigpetrov** [comment]
Наши решения работают по реестру блокировок РКН,
Мы планируем вести список внереестровых блокировок, на ТСПУ (тоже от РКН).

Сайты, ограничивающие доступ по собственной инициативе, в эти два реестра не попадают.
Таких сайтов довольно много -- будет очень трудозатратно вести их реестр, потому мы этого не делаем и не планируем.
Добавлять Тор или свои прокси -- рекомендуемое решение для таких случаев.
